CHANGELOG.md

```diff
@@ -2,224 +2,256 @@
 
 ## [Unreleased]
 
-* Changes
+### Changed
```

Comment:
I'm nit-picking here, but for general markdown linting, let's follow headings with a single blank line.
```suggestion
### Changed

```